test/qbits/spandex/test/core_test.clj

```diff
@@ -16,27 +17,24 @@
   (catch Exception e
     (.printStackTrace e)))
 
-(def server "http://127.0.0.1:9200")
+(def server "http://127.0.0.1:9250")
```

Comment:
This will break Travis CI build as the build uses the default 9200 port. Btw @mpenet shouldn't the travis build be ran on pull requests too?

Reply:
yup debug things, fixed. Thank you.